### DISCUSSION TITLE
H3 40xxx registers

### DISCUSSION THREAD

**Discussion by canton7** [TonyM1958/HA-FoxESS-Modbus#10] [Q&A]
Hi,

I notice you've got 40xxx registers listed for the H3, which aren't in StealthChesnet's original YAML file. Have those been tested?

Thanks!

**Comment by TonyM1958** [comment]
They are there so they can be tested if we find someone with an H3 to check them on. I only read them to get values.

If you know someone with H3, ask them to probe for values.

In practice, there isn't a lot here that you would really want to report on.

**Comment by kinkate** [comment]
Thank you Tony for your Modbus Map.xls! A few friends and I from a German PV forum tried the remote control of an "H3" inverter and your table helped a lot. Currently there are some changes by FoxEss Inc. : The "PowerStation" working mode is no longer available in the official FoxEssCloudControl app, but the "PowerShaving" mode is described in detail (with sketches!). Whatever FoxEss Inc. is trying to do with it. There are a few people on this forum who have tested changing the 41xxx registers.
